content/en/code_analysis/_index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Code Analysis
+kind: documentation
+description: Learn how to use Datadog Code Analysis to address maintainability issues, bugs, and security vulnerabilities in development to prevent customer impact.
+is_beta: true
+further_reading:
+- link: "https://www.datadoghq.com/blog/monitor-ci-pipelines/"
+  tag: "Blog"
+  text: "Monitor all your CI pipelines with Datadog"
+- link: "/integrations/guide/source-code-integration/"
+  tag: "Documentation"
+  text: "Learn about the Source Code Integration"
+- link: "/code_analysis/static_analysis"
+  tag: "Documentation"
+  text: "Learn about Static Analysis"
+- link: "/code_analysis/software_composition_analysis"
+  tag: "Documentation"
+  text: "Learn about Software Composition Analysis"
+---
+
+{{% site-region region="gov" %}}
+<div class="alert alert-danger">
+    Code Analysis is not available for the {{< region-param key="dd_site_name" >}} site.
+</div>
+{{% /site-region %}}
+
+{{< callout url="#" btn_hidden="true" header="Try the Beta!" >}}
+Code Analysis is in public beta.
+{{< /callout >}}
+
+## Overview
+
+Code Analysis is composed of the [Static Analysis][1] and [Software Composition Analysis (SCA)][2] products. 
+
+Static Analysis
+: Scans your bespoke code for maintainability issues, bugs, performance issues, and security vulnerabilities early in the development lifecycle to catch issues from reaching production and, when possible, provide suggested fixes to help engineering teams address these issues before they impact customers.
+
+Software Composition Analysis 
+: Scans the open-source libraries that are imported into your repositories for known vulnerabilities. Coming soon will be the ability to block libraries with certain licenses and suggestions on which version of a library you should update to so that you can resolve the vulnerability.
```

Comment:
```suggestion
: Scans the open-source libraries that are imported into your repositories for known vulnerabilities. In a future release, you will have the ability to block libraries with certain licenses, and apply suggestions to upgrade to remediated library versions, so that you can resolve the vulnerability.
```
I know we typically avoid words as "coming soon" , I think we should change this to "in a future release" ?

Reply:
Can you fix open-source here too, to "open source" ?